packages/tailwindcss/src/compat/plugin-api.ts

```diff
@@ -17,7 +17,7 @@ import * as SelectorParser from './selector-parser'
 
 export type Config = UserConfig
 export type PluginFn = (api: PluginAPI) => void
-export type PluginWithConfig = { handler: PluginFn; config?: UserConfig }
+export type PluginWithConfig = { handler: PluginFn; config?: UserConfig; reference?: boolean }
```

Comment:
```suggestion
export type PluginWithConfig = { 
  handler: PluginFn;
  config?: UserConfig;
  
  /** @internal */
  reference?: boolean
}
```

Reply:
Think we should mark it as internal in case we figure out a way to thread the information through the system without exposing it at all.